examples/predicted_molecular_consequences/rs699.json

```diff
@@ -0,0 +1,260 @@
+                "url": "www.sequenceontology.org",
+                "description": "The Sequence Ontology..."
+            }
+            },
+
+            "consequences":
+            [{
+            "accession_id": "SO:0001583",
+            "value": "missense_variant",
+            "url": "www.sequenceontology.org/browser/current_release/term/SO:0001583",
+            "source": {
+                "id": "...",
+                "name": "Sequence Ontology",
+                "url": "www.sequenceontology.org",
+                "description": "The Sequence Ontology..."
+            }
+            },
+            {
+                "accession_id": "SO:0002114",
+                "value": "NMD_transcript_variant",
+                "url": "www.sequenceontology.org/browser/current_release/term/SO:0002114",
+                "source": {
+                    "id": "...",
+                    "name": "Sequence Ontology",
+                    "url": "www.sequenceontology.org",
+                    "description": "The Sequence Ontology..."
+                }
+            }
+            ],
+            "gene_stable_id" : "ENSG00000135744",
+            "gene_symbol": "AGT",
+            "protein_stable_id": null,
+            "prediction_results": [{   
+                                        "score": 1  ,
+                                        "result": null,
+                                        "classification": "tolerated",
+                                        "analysis_method" : { 
+                                                                "tool": "SIFT"
+                                                            }
+                                    },
+                                    {
+                                          
+                                        "score": 0  ,
+                                        "result": null,
+                                        "classification": "benign",
+                                        "analysis_method" : { 
+                                                                "tool": "PolyPhen"
+                                                            }
+                                        
+                                    }
+                                ],
+            "transcript_biotype": "nonsense_mediated_decay",
+            "cdna_location": 
+            {
+                "relation": "overlaps" ,
+                "start": 1287,
+                "end": 1287, 
+                "length": 1, 
+                "percentage_overlap": 100 ,
+                "ref_sequence": "T",
+                "alt_sequence": "C"
+            },
+            "cds_location": 
+            {
+                "relation": "overlaps" ,
+                "start": 776, 
+                "end": 776, 
+                "length": 1, 
+                "percentage_overlap": 100,
+                "ref_sequence": "ATG",
+                "alt_sequence": "ACG"
```

Comment:
I've just realised that you are reporting different sequences for cDNA and CDS. It looks like for CDS, you are reporting triplets. But:

- This contradicts the value of the `length` parameter, which says `1`
- What does "start" refer to? what does "end" refer to? It seems like both fields refer to the position of the changed nucleotide; yet according to the `ref_sequence`/`alt_sequence` field, this nucleotide is second in the reported triplet. How can the client know this? 
- In Andrea's [design](https://xd.adobe.com/view/573e7994-6561-412f-84f0-b34c0d38761e-026d/screen/e578a382-b0c0-40c9-b2d9-05a7007adafc), she has marked a single nucleotide change. Is this different from your expectations, according to the above example data?

![image](https://github.com/Ensembl/ensembl-vdm-docs/assets/6834224/c011e965-153d-464c-ad29-53db6b2d2310)

Reply:
This is currently not priority, created a [ticket](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-6239) to handle it